tests/Gedmo/Sortable/Fixture/CustomerType.php

```diff
@@ -88,9 +91,20 @@ public function removeCustomer(Customer $customer)
     public function postRemove()
     {
         if ($this->getCustomers()->count() > 0) {
-            // we imitate an foreign key constraint exception, because doctrine
-            // does not support sqlite constraints, which must be tested, too.
+            // we imitate a foreign key constraint exception because Doctrine
+            // does not support SQLite constraints, which must be tested, too.
+
             $pdoException = new \PDOException('SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails', '23000');
+
+            // doctrine/dbal 3.x changes the signature of the `DriverException::__construct()`.
+            $refl = new \ReflectionMethod(DriverException::class, '__construct');
```

Comment:
Isn't simpler to check against the existence of the `Doctrine\DBAL\Driver\PDOException` class in order to determine the installed `doctrine/dbal` version?